src/pages/RunDetails.test.js

```diff
@@ -181,6 +183,10 @@ describe('RunDetails - render smoke test', () => {
         expect(stdout).toBeDefined
         const stderr = within(row).getByText('stderr')
         expect(stderr).toBeDefined
+        const inputs = within(row).getByText('Inputs')
+        expect(inputs).toBeDefined
+        const outputs = within(row).getByText('Outputs')
+        expect(outputs).toBeDefined
```

Comment:
This kind of testing seems sensitive to any kind of localization or copy changes and thus seems too closely coupled  (for my personal subjective standard)

Reply:
Makes sense to me! I'll update a few tests to use testids instead.